core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-add-variable/dot-add-variable.component.ts

```diff
@@ -15,13 +17,23 @@ import { FilteredFieldTypes } from './dot-add-variable.models';
     providers: [DotAddVariableStore]
 })
 export class DotAddVariableComponent implements OnInit {
-    vm$ = this.store.vm$.pipe(
+    vm$: Observable<DotVariableList> = this.store.vm$.pipe(
         map((res) => {
-            const variables = res.variables.filter(
-                (variable) =>
-                    variable.fieldType !== FilteredFieldTypes.Column &&
-                    variable.fieldType !== FilteredFieldTypes.Row
-            );
+            const variables: DotVariableContent[] = res.variables
+                .filter(
+                    (variable) =>
+                        variable.fieldType !== FilteredFieldTypes.Column &&
+                        variable.fieldType !== FilteredFieldTypes.Row
+                )
+                .map((variable) => ({
+                    name: variable.name,
+                    variable: variable.variable,
+                    fieldTypeLabel: variable.fieldTypeLabel
+                }));
+            variables.push({
+                name: 'Content Identifier Value',
```

Comment:
This needs to be translated, I am pretty sure there is a language key for this.

Reply:
mm How i can push this variable as translatable item?
Also, in the html, the variable name and fieldTypeLabel arent translated, they are only pushed 
<img width="578" alt="Screenshot 2023-09-27 at 12 24 35" src="https://github.com/dotCMS/core/assets/144152756/61c2461a-5ec3-4ee0-af4a-f62ee711f607">

Reply:
Yes in that case we don't need to translate the `name` or `fieldTypeLabel` because those are user input. But `Content Identifier Value` is a hardcoded name, we need to translate it. For that we have `DotMessageService` to use it you just need to find the correct key in the `Language.Properties` I think the key is `Content-Identifier-value` you can double check to be sure.